pkg/webhooks/regularuser/regularuser_test.go

```diff
@@ -280,6 +280,39 @@ func TestNodesSubjectPermissionsClusterVersions(t *testing.T) {
 			operation:       v1beta1.Delete,
 			shouldBeAllowed: false,
 		},
+		{
+			testID:          "schedulers-admin-user",
+			targetResource:  "schedulers",
+			targetKind:      "Scheduler",
+			targetVersion:   "v1",
+			targetGroup:     "",
+			username:        "kube:admin",
+			userGroups:      []string{"system:authenticated", "system:authenticated:oauth"},
+			operation:       v1beta1.Update,
+			shouldBeAllowed: true,
+		},
+		{
+			testID:          "schedulers-priv-user",
+			targetResource:  "schedulers",
+			targetKind:      "Scheduler",
+			targetVersion:   "v1",
+			targetGroup:     "",
+			username:        "my-user",
```

Comment:
I see my-name vs my-user in the structures for some tests - I assume this doesn't matter?

Reply:
Ah, yes - I see it in other places; Looks fine

Reply:
Yeah, the only significance is that it's not a "special" username like `kube:admin`.